Tools/Test.UnitTests.Tools/Cwe/CweTaxonomyGeneratorTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif;
+
+using Newtonsoft.Json;
+
+using Taxonomy.Cwe;
+
+using Xunit;
+
+namespace Test.UnitTests.Tools.Cwe
+{
+    public class CweTaxonomyGeneratorTests
+    {
+        [Theory]
+        [InlineData("4.3", "2020-12-10", 1335)]
```

Comment:
>InlineData [](http://example.com/codeflow?start=9&length=10)

consider replacing the Theory InlineData tests with Fact tests

Reply:
Yes! Fixed.